### PR TITLE
neonvm: add LFC approximate working set size to metrics

### PR DESCRIPTION
**Pull request by skyzh**
## Problem

ref https://github.com/neondatabase/autoscaling/pull/878
ref https://github.com/neondatabase/autoscaling/issues/872

## Summary of changes

Add `approximate_working_set_size` to sql exporter so that autoscaling can use it in the future.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

**Comment by skyzh** [comment]
do not merge until https://github.com/neondatabase/neon/pull/7231 gets merged

File: vm-image-spec.yaml
Comment (Bodobolero) [inline vm-image-spec.yaml:192]:
```suggestion
        help: 'Approximate working set size in pages of 8192 bytes'
```

**Review by Bodobolero — APPROVED** [review]
We have clarified in slack that we need to make sure that neon extension in compute VM is upgraded to latest version BEFORE this PR is merged and deployed in prod.
If they are deployed together, we will have some ephemeral errors in compute log at the first compute start after new image is in prod, but after that no more, which is fine.

**Review by zaynetro — APPROVED** [review]

**Pull request merged**
